app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -339,7 +339,16 @@ def parse_container_definition(container_def, pod_id)
         :image_pull_policy => container_def.imagePullPolicy,
         :memory            => container_def.memory,
          # https://github.com/GoogleCloudPlatform/kubernetes/blob/0b801a91b15591e2e6e156cf714bfb866807bf30/pkg/api/v1beta3/types.go#L815
-        :cpu_cores         => container_def.cpu.to_f / 1000
+        :cpu_cores         => container_def.cpu.to_f / 1000,
+        :capabilities_add  => container_def.securityContext.try(:capabilities).try(:add).to_s,
```

Comment:
`to_s` will produce a string like: `["CAP_ONE", "CAP_TWO", "CAP_THREE"]` I think that a comma separated list is better:

    container_def.securityContext.try(:capabilities).try(:add).to_a.join(',')

which would produce:

    CAP_ONE,CAP_TWO,CAP_THREE